.idea/misc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="ES6" />
+  </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="RVM: ruby-2.2.0" project-jdk-type="RUBY_SDK" />
```

Comment:
I would recommend adding the `.idea/` folder to `.gitignore`. Settings like this one would likely conflict with settings other developers use in their Rubymine setups. For example, my project jdk wouldn't work because I don't have RVM, also, I'm using a much newer version of Ruby.